src/content/en/updates/2018/03/present-web-pages-to-secondary-attached-displays.md

```diff
@@ -0,0 +1,268 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Present web pages that support the Presentation API to a secondary attached displays.
+
+{# wf_updated_on: 2018-03-27 #}
+{# wf_published_on: 2018-03-27 #}
+{# wf_tags: news,media,presentation #}
+{# wf_featured_image: /web/updates/images/2018/03/present-web-pages-to-secondary-attached-displays/hero.jpg #}
+{# wf_featured_snippet: Chrome 66 allows users to present web pages that support the Presentation API to a secondary attached display, and control the contents being shown from the controlling page on the main display with the Presentation Receiver API. #}
```

Comment:
Suggested rewrite:

Chrome 66 allows web pages to use a secondary attached display through the Presentation API and to control its contents through the Presentation Receiver API.

I'd also suggest copying this rewrite as is to the description and to the lead paragraph at line 16.

Reply:
Done.